pkg/federation/get.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2023 The Bestchains Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package federation
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/bestchains/bc-cli/pkg/common"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubectl/pkg/cmd/get"
+)
+
+func NewFedGetCmd(option common.Options) *cobra.Command {
```

Comment:
1. get all federations that current user joins
```shell
bc-cli get feds
```
2. get a specific federation
```shell
bc-cli get fed {fed-name}
```
support:
- `-oyaml`
- `-ojson`

3. get  allfederation that belongs to a specific org
```shell
bc-cli get feds  --with-org {org_name}
```